src/components/DomEventsTable.js

```diff
@@ -0,0 +1,204 @@
+import React, { useRef, useState, useCallback } from 'react';
+import IconButton from './IconButton';
+import TrashcanIcon from './icons/TrashcanIcon';
+import EmptyStreetImg from '../images/EmptyStreetImg';
+import StickyList from './StickyList';
+import { VirtualScrollable } from './Scrollable';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import CopyButton from './CopyButton';
+import Select from 'react-select';
+
+const defaultNamesOptions = [
+  { label: 'mouseEnter', value: 'mouseEnter' },
+  { label: 'mouseMove', value: 'mouseMove' },
+  { label: 'mouseOver', value: 'mouseOver' },
+  { label: 'mouseOut', value: 'mouseOut' },
+  { label: 'mouseLeave', value: 'mouseLeave' },
+  { label: 'pointerOver', value: 'pointerOver' },
+  { label: 'pointerMove', value: 'pointerMove' },
+  { label: 'pointerEnter', value: 'pointerEnter' },
+  { label: 'pointerLeave', value: 'pointerLeave' },
+  { label: 'pointerOut', value: 'pointerOut' },
+];
+
+function EventRecord({ index, style, data }) {
+  const { id, event, target } = data[index];
+
+  return (
+    <div
+      className={`w-full h-10 flex items-center text-sm ${
+        index % 2 ? 'bg-gray-100' : ''
+      }`}
+      style={style}
+    >
+      <div className="p-2 flex-none w-16">{id}</div>
+
+      <div className="p-2 flex-none w-40">{event.EventType}</div>
+      <div className="p-2 flex-none w-40">{event.name}</div>
+
+      <div className="p-2 flex-none w-40">{target.tagName}</div>
+      <div className="p-2 flex-auto whitespace-no-wrap">
+        {target.toString()}
+      </div>
+    </div>
+  );
+}
+
+function SelectHeader({ label, options, onChange, value }) {
+  const valueContainer = () => label;
+
+  const styles = {
+    control: (style) => ({
+      ...style,
+      paddingLeft: '0.5rem',
```

Comment:
You may also want to add a bit of `padding-bottom`

<img width="940" alt="Screen Shot 2020-06-21 at 11 11 55 AM" src="https://user-images.githubusercontent.com/20029120/85228200-0ce26100-b3b0-11ea-9402-2a526c6a5332.png">

Reply:
Thanks @sumeesh879! It wasn't an issue related to a missing padding but to a wrong height in the header row and I fixed it!
Thanks!